test/cli/page_width_test.dart

```diff
@@ -50,40 +47,20 @@ void main() {
         d.file('main.dart', _unformatted),
       ]).create();
 
-      var process = await runFormatterOnDir([
-        '--enable-experiment=tall-style',
-        '--line-length=30',
-      ]);
+      var process = await runFormatterOnDir(['--line-length=30']);
       await process.shouldExit(0);
 
       await d.dir('foo', [d.file('main.dart', _formatted30)]).validate();
     });
   });
 
-  test('no options search if experiment is off', () async {
-    await d.dir('foo', [
-      analysisOptionsFile(pageWidth: 20),
-      d.file('main.dart', _unformatted),
-    ]).create();
-
-    var process = await runFormatterOnDir();
-    await process.shouldExit(0);
-
-    // Should format the file at the default width.
-    await d.dir('foo', [d.file('main.dart', _formatted80)]).validate();
-  });
-
   test('no options search if page width is specified on the CLI', () async {
     await d.dir('foo', [
       analysisOptionsFile(pageWidth: 20),
       d.file('main.dart', _unformatted),
     ]).create();
 
-    var process = await runFormatterOnDir([
-      '--language-version=latest', // Error to not have language version.
```

Comment:
These `--language-version=latest` lines were no longer necessary as of #1573, but I forgot to clean them up in that PR.